cli/cmd/plugin/standalone-cluster/tanzu/tanzu.go

```diff
@@ -171,8 +171,14 @@ func (t *TanzuLocal) Deploy(lcConfig *config.LocalClusterConfig) error {
 	log.Event("\\U+1F4E6", "Selected kapp-controller image bundle\n")
 	log.Style(outputIndent, logger.ColorLightGrey).Infof("%s\n", t.kappControllerBundle.GetRegistryURL())
 
-	// 4. Create the cluster
-	log.Eventf("\\U+1F6F0", " Creating cluster %s\n", lcConfig.ClusterName)
+	// 4. Create the cluster)
+	log.Eventf("\\U+1F4E5", "Pulling base image %s\n", lcConfig.NodeImage)
```

Comment:
Can we make this log message more consistent to the rest?

![image](https://user-images.githubusercontent.com/6200057/139703419-9d58ebe0-b1b3-4538-b1b1-d159b3134bf9.png)

My intuition is something like:

```txt
📥  Pulling base image 
      projects.registry.vmware.com/tce/kind/node:v1.21.5
      Pulling..... <--- these tick
```

Alternatively, I also think this could go very well under the `Creating cluster` log message:

```txt
🛰 Creating cluster doggo
     Pulling image.... <-- these tick
     To troubleshoot, use:
     kubectl ${COMMAND} --kubeconfig /home/josh/.config/tanzu/tkg/local/doggo/kube.conf
```

Reply:
I like the idea of just having it under the "Creating cluster" step. 👍

Reply:
For now at least, not making the `...` animated to show progress due to the added complexity it would take to make these synchronous calls async. We can certainly do it though. If you'd like that done now I will get it added, but otherwise this will now at least show the line so there is some indication to the user that there is something being done.

Reply:
> the added complexity it would take to make these synchronous calls async.

I was thinking the same thing - it probably warrants a separate PR that creates go routines to log those animations based on a `context`